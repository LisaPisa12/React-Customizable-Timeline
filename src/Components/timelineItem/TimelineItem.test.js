import React from 'react';
import { render } from '@testing-library/react';
import ReactDOM from 'react-dom';
import TimelineItem from './TimelineItem';

it('matches snapshot', () => {
  const container = document.createElement('div');
  ReactDOM.render(<TimelineItem />, container);
  expect(container).toMatchSnapshot();
});
