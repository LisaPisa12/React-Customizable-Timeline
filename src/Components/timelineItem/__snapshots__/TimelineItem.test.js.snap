// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="sc-bxivhb SUHGw"
  >
    <div>
      <div
        style="cursor: pointer;"
      >
        <h4 />
        <h5 />
      </div>
    </div>
  </div>
</div>
`;
